app/serializers/external_registry_bike_v3_serializer.rb

```diff
@@ -0,0 +1,63 @@
+class ExternalRegistryBikeV3Serializer < BikeV2Serializer
+  delegate \
+    :external_id,
+    :location_found,
+    :status,
+    :url,
+    to: :object,
+    allow_nil: true
+
+  def stolen
+    object.stolen?
+  end
+
+  def status
+    object.status&.titleize
```

Comment:
I don't think you need to declare this, if the only purpose is titleizing. I think you can skip this method. Ditto for `mnfg_name` and `frame_model`

Reply:
> I don't think you need to declare this

In the sense of:
(a) the strings in question are already titleized, or 
(b) we shouldn't bother titleizing them for the UI?  🤔

Reply:
Touché.

Are things capitalized automatically for the other serializer though? If not, it feels weird

Reply:
> Are things capitalized automatically for the other serializer though? If not, it feels weird

mm true. I think that suggests (c) — I'll move the titleizing to the React component.

Reply:
Updated in bc2ab9ac